### PR TITLE
NEPT-2221: HTML tags in Webtools JSON Object field

### PR DESCRIPTION
**Pull request by enriquelacoma**
### Description
Allow tags `<b>, <strong>, <i>, <em>, <br/>, <span>` on the json filter

### Change log

- Changed: filter function of webtools render

### CODE REVIEW

File: profiles/common/modules/custom/nexteuropa_json_field/nexteuropa_json_field.module [hunk 1]
```diff
@@ -125,7 +125,8 @@ function nexteuropa_json_field_field_formatter_view($entity_type, $entity, $fiel
       foreach ($items as $delta => $item) {
         $element[$delta] = array();
         // Strip_tags to avoid javascript injection.
```
Comment (ku-enza) [inline profiles/common/modules/custom/nexteuropa_json_field/nexteuropa_json_field.module]: As you removes strip_tags function, and uses filter_xss, can you also Update  this comment line?

**Review by ku-enza — CHANGES_REQUESTED** [review]

File: profiles/common/modules/custom/nexteuropa_json_field/nexteuropa_json_field.module (hunk 1 above)
Comment (enriquelacoma) [reply profiles/common/modules/custom/nexteuropa_json_field/nexteuropa_json_field.module]: Comments updated

**Review by ku-enza — APPROVED** [review]
Ok for me.

**Review by Fefaine — APPROVED** [review]
OK

**Pull request merged**